packaging/linux/deb/debroot.go

```diff
@@ -296,6 +296,16 @@ func fixupGenerators(spec *dalec.Spec, cfg *SourcePkgConfig) []byte {
 		fmt.Fprint(buf, "\n")
 	}
 
+	if spec.HasPips() {
+		// Set PIP environment variables to point to our prepared pip packages
+		// Use --break-system-packages since we're in an isolated build container
```

Comment:
Was there a specific reason you needed to set `break-system-packages` here? In this case our "system" would be the isolated build container, it's true, but I'm not sure that necessarily means we couldn't run into conflicts. I'm wondering about the case where someone has a build dependency (installed via the package manager) that depends on python + python packages. If they then use the pip generator, could that cause issues?